Slack.Automation/Promact.Erp.Util/EnvironmentVariableRepository/EnvironmentVariableRepository.cs

```diff
@@ -67,11 +67,11 @@ public string SlackOAuthClientId
             }
         }
 
-        public int Port
+        public string Port
```

Comment:
Why would you change it to string again? Port will always be an integer then why is it in string?